study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -122,8 +122,7 @@ private void run(String[] args) throws Exception {
         log("using study configuration file: %s", cfgPath);
 
         log("resolving study configuration...");
-        studyCfg = studyCfg.resolveWith(varsCfg);
-
+        studyCfg = studyCfg.withFallback(varsCfg).resolve();
```

Comment:
Local substitutions now needed for study.conf file processing.

Reply:
Do you think we can revert this back to `.resolveWith(varsCfg)`? It _sounds_ like it'll do the same thing.

Reply:
I also would have thought that this change would have been unnecessary, but I was forced to make it after introducing the substitutions in the .conf file and having study builder trip over them.